### DISCUSSION TITLE
Question regarding FPA

### DISCUSSION THREAD

**Discussion by gus-kfz** [jilleb/MQB-FPA#88] [Q&A]
Hi,

sorry for my question, i'm new to these stuff, but I don't understand how to activate the FPA profiles and which files and how to write on the Gateway? is it only Dataset which i can write it normally by VCP/ODIS or I must use another software?

How about swap codes here? if I'm correct, then FPA is secured with swap codes, how it will be done?

Thank you so much for your support!

**Comment by jilleb** [comment]
Hi!
It's not possible to enable FPA when you dont have the swap codes/FECs for this functionality.
